package.json

```diff
@@ -140,5 +140,9 @@
       "@changesets/get-dependents-graph@1.3.6": "patches/@changesets__get-dependents-graph@1.3.6.patch",
       "@changesets/assemble-release-plan@5.2.4": "patches/@changesets__assemble-release-plan@5.2.4.patch"
     }
+  },
+  "devDependencies": {
+    "typedoc": "^0.25.13",
+    "typescript": "^5.4.5"
```

Comment:
I think in the root monorepo we are safe to put everything in `dependencies` since end users never use this